types/twine-sugarcube/test/settings.ts

```diff
@@ -0,0 +1,42 @@
+Setting.addHeader("head"); // $ExpectType void
+Setting.addHeader("head", "headDesc"); // $ExpectType void
+Setting.addToggle("toggle", {
+	label: "a toggle",
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/test/settings.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.